src/ios/ESMarker.h

```diff
@@ -6,10 +6,12 @@
 //
 //
 
+#import "ESMap.h"
 #import <Tabris/BasicWidget.h>
 #import <MapKit/MapKit.h>
 
 @interface ESMarker : BasicWidget <MKAnnotation>
+@property (strong) ESMap *map;
```

Comment:
I wonder if this `strong` reference will not introduce a retain cycle, while annotations are already stored probably in an array by `MKMapView`. I see that you clear the reference in `-destroy` method, so if the object will be destroyed from JS it will work fine.

However, in any other case, it will not, we could think about the other cases, what they are? Or we could change it to `weak`, it should not make a difference atm.

BTW if we decide to change it, you can remove `-destroy` method while it's not needed.

What do you think?